package.json

```diff
@@ -4,7 +4,7 @@
     "description": "PayPal Checkout components, for integrating checkout products.",
     "main": "index.js",
     "scripts": {
-        "dev": "babel-node $(npm bin)/webpack-dev-server --config webpack.config.dev.js --port 9001 --host localhost.paypal.com --open-page demo/dev/index.htm --https --hot=false --inline=false",
+        "dev": "NODE_ENV=test babel-node $(npm bin)/webpack-dev-server --config webpack.config.dev.js --port 9001 --host localhost.paypal.com --open-page demo/dev/index.htm --https --hot=false --inline=false",
```

Comment:
Can we remove this? I don't think we should run dev env in test mode.

Reply:
@mnicpt I took a look at the webpack config and I think we can remove `NODE_ENV=test` from here. And to fix the error, we can add `__PAYPAL_DOMAIN__` and give it a proper value for local development: https://github.com/paypal/paypal-checkout-components/blob/master/webpack.config.dev.js#L30
```
__PAYPAL_DOMAIN__:   'https://www.sandbox.paypal.com',
```